src/tests.js

```diff
@@ -1,3 +1,15 @@
+import chai from 'chai';
+import sinonChai from 'sinon-chai';
+import chaiEnzyme from 'chai-enzyme';
+import chaiAsPromised from 'chai-as-promised';
+import sinon from 'sinon';
+import sinonStubPromise from 'sinon-stub-promise';
+
+chai.use(sinonChai);
+chai.use(chaiEnzyme());
+chai.use(chaiAsPromised);
+sinonStubPromise(sinon);
+
 // load all tests into one bundle
 const testsContext = require.context('.', true, /\.test\.js$/);
```

Comment:
Use es6 import instead of require.

Reply:
I don't think it is possible to use `import` for this task. If you know how, please tell me.

Reply:
you can try this https://github.com/crisberrios/dir-loader

Reply:
I looked at `dir-loader`, but it is still using `require`, so I don't see how does it help. 

I don't see what is the problem with the current solution.